docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -483,6 +483,127 @@ The TransactionManager controls the timeout of database queries.
 
 To modify the default transaction timeout value, set the environment variable `FHIR_TRANSACTION_MANAGER_TIMEOUT` or enter the value in the server.env file at the root of the WLP fhir-server instance. Example values are `120s` (seconds) or `2m` (minutes).
 
+### 3.3.2 Payload Offload for Azure Blob
+
+The LinuxForHealth FHIR Server can be configured to store the resource payload records in an Azure Blob container. Each record is stored as a JSON string with UTF-8 encoding. The LinuxForHealth FHIR Server relies on the blob service to compress/encrypt the data at rest. The blob service must apply the necessary security controls required when storing PHI, and connection to the service must use an encrypted transport.
+
+#### 3.3.2.1 Payload Offload Configuration for Azure Blob
+
+To enable payload offloading using Azure Blob storage, complete the steps summarized below:
+
+1. Pick a container name to use for each tenant/datasource combination;
+2. Create a database.properties file with connection information for the FHIRSERVER user (not FHIRADMIN);
+3. Configure payload offloading in the main `default/fhir-server-config.json` file;
+4. Configure payload offloading in each tenant-specific `fhir-server-config.json` file;
+5. Create the container using the Azure Blob user interface, the Azure Blob API or the LinuxForHealth FHIR Server fhir-persistence-blob-app-*-cli.jar tool;
+6. (Optional) Periodically run the reconciliation tool to remove any orphan resource records from the offload datastore. See the fhir-persistence-blob-app project [README](https://github.com/LinuxForHealth/FHIR/tree/main/fhir-persistence-blob-app/README.md) for more information.
+
+Take the following restrictions into account:
+
+1. Payload offloading is enabled at the server level. Offloading cannot be enabled/disabled on a per-tenant basis. If you want to support offloading for just one tenant, use a different IBM FHIR Server instance;
+2. Payload offloading must be configured prior to ingesting any resource data;
+3. Payload offloading must not be disabled after resource data has been ingested;
+4. When payload offloading is enabled, FHIR resources are limited to 5000 MiB in size when rendered as uncompressed JSON. The practical limit is likely to be lower due to memory usage and typical socket and transaction timeouts.
+
+
+#### 3.3.2.2 Pick a Container Name
+
+Each tenant/datasource combination requires its own container. Use a name that can be easily identified as belonging to the tenant and datasource, subject to the Azure Blob service naming restrictions.
```

Comment:
a link to said naming restrictions might be nice...i recall there being at least one gotcha

Reply:
i found it below...should be OK